Fix bug in battery status

ela-kotulska-frequenz opened the pull request:
Inverter that was discharging was considered as not working.
If inverter was discharging, then it was impossible to
set any command until it changed state.

ela-kotulska-frequenz committed:
Format code with black

Signed-off-by: ela-kotulska-frequenz <elzbieta.kotulska@frequenz.com>

leandro-lucarella-frequenz commented on pyproject.toml:
```diff
@@ -28,7 +28,7 @@ dependencies = [
     "frequenz-channels >= 0.11.0, < 0.12.0",
     "google-api-python-client >= 2.71, < 3",
-    "grpcio >= 1.51.1, < 2",
+    "grpcio == 1.51.1",
```
Can you maybe split all the pinning (here and in `noxfile.py`) to a separate commit to make it more clear (I guess this is to make the CI pass?).

We probably should consider start pinning the dev dependencies always, so we don't have to do this when we need to release a hot fix for an old version.

ela-kotulska-frequenz replied:
True :) Thanks. I separated commits.

leandro-lucarella-frequenz reviewed (dismissed):
Just a suggestion, but feel free to merge as is too it is it too much work, it is an old release anyway...

ela-kotulska-frequenz committed:
Pin some dev dependencies

..Otherwise CI will fail.
pylint / mypy - because version was upgraded and tools find
much more linting erros.
grpcio - because we use it in our unit tests in wrong way.
And with the newer version our unit tests will fail.
This is problem with unit tests that was fixed in newer
release.

Signed-off-by: ela-kotulska-frequenz <elzbieta.kotulska@frequenz.com>

ela-kotulska-frequenz committed:
Fix bug in BatteryStatus

Inverter that was discharging was considered as not working.
If inverter was discharging, then it was impossible to
set any other command until it changed state.

Signed-off-by: ela-kotulska-frequenz <elzbieta.kotulska@frequenz.com>
